app/views/documents/history.html.erb

```diff
@@ -67,6 +73,16 @@
         <% end %>
       </div>
     <% end %>
+
+    <% pages = { } %>
+
+    <% pages[:previous_page] = { url: history_path_url(page: @timeline_entries.prev_page),
+                                 title: "Previous page" } if @timeline_entries.prev_page %>
+
+    <% pages[:next_page] = { url: history_path_url(page: @timeline_entries.next_page),
+                             title: "Next page" } if @timeline_entries.next_page %>
```

Comment:
We can probably put this into a single block of embedded ruby:

<%
pages = {}
pages[:previous_page] = { url: history_path_url(page: @timeline_entries.prev_page),
                                 title: "Previous page" } if @timeline_entries.prev_page
pages[:next_page] = { url: history_path_url(page: @timeline_entries.next_page),
                             title: "Next page" } if @timeline_entries.next_page
%>